_posts/2020-03-24-dask-developer-workshop.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: single
+title:  "Tales from D.C.: Dask Developer Workshop 2020"
+date:   2020-03-24 20:00:00 +0100
+tags: technology python data-engineering dask distributed
+header:
+  overlay_image: assets/images/tech_gear_banner.jpg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Lucas Rademaker
+author_profile: true
+---
+# Tales from D.C.: Dask Developer Workshop 2020
+
+## Setting
+Back in January this year, [Dask](https://dask.org/) community members received an invite for a workshop in Washington DC with the goal of "connecting advanced users with developers".
+
+## The workshop
```

Comment:
Maybe you can merge the `Setting` and the `The workshop` section to one? Both feel a bit like an introduction.